### PR TITLE
fix:Readme.md added

### PR DESCRIPTION
**Pull request by Piyuse**
Added the readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
+## Contributing
+
+If you would like to contribute, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch.
+3. Make your changes.
+4. Commit your changes.
+5. Push to the branch
+6. Create a new Pull Request.
```
Comment (mellofordev) [inline README.md]: please remove this

File: README.md [hunk 2]
```diff
@@ -1,27 +1,49 @@
...
 npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
 ```
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```
Comment (mellofordev) [inline README.md]: remove this line

**Review by mellofordev — CHANGES_REQUESTED** [review]
remove: 
from line 36 - 39 ,
from line 40 - 49

File: README.md (hunk 1 above)
Comment (Piyuse) [reply README.md]: ok

**Review by mellofordev — APPROVED** [review]
okay accepting this

**Pull request merged**